purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -753,6 +757,37 @@ internal class BillingWrapper(
 
     override fun isConnected(): Boolean = billingClient?.isReady ?: false
 
+    override fun showInAppMessagesIfNeeded(activity: Activity) {
+        val inAppMessageParams = InAppMessageParams.newBuilder()
+            .addInAppMessageCategoryToShow(InAppMessageParams.InAppMessageCategoryId.TRANSACTIONAL)
+            .build()
+        val weakActivity = WeakReference(activity)
```

Comment:
I'm using a weak reference here since I want to avoid any chance of leaking the activity if there is a problem connecting to the billing client.